src/main/java/com/mrh0/createaddition/blocks/modular_accumulator/ModularAccumulatorTileEntity.java

```diff
@@ -485,7 +481,7 @@ public void notifyMultiUpdated() {
 		setChanged();
 		// When the multi block is updated, the neighborChanged method isn't fired,
 		// so update the cache here instead.
-		updateCache();
+		//updateCache();
```

Comment:
Any reason this is commented out?

Reply:
Yeah, it shouldn't be needed anymore as I changed the flags for the setBlock last PR if I remember correctly, I just forgot to comment it out at that time.

Block.UPDATE_NEIGHBORS should do the same thing.
```java
level.setBlock(getBlockPos(), state, Block.UPDATE_NEIGHBORS | Block.UPDATE_CLIENTS | Block.UPDATE_INVISIBLE);
```